src/cdk-experimental/testing/protractor/protractor-element.ts

```diff
@@ -6,8 +6,42 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
-import {browser, ElementFinder} from 'protractor';
-import {TestElement} from '../test-element';
+import {ModifierKeys} from '@angular/cdk/testing';
+import {browser, ElementFinder, Key} from 'protractor';
+import {TestElement, TestKey} from '../test-element';
+
+const keyMap = {
```

Comment:
Should we have a comment for this constant to say what it is responsible for? (same for the test bed implementation)